pkg/pillar/cmd/diag/diag.go

```diff
@@ -20,24 +28,11 @@ import (
 	"github.com/lf-edge/eve/pkg/pillar/types"
 	"github.com/lf-edge/eve/pkg/pillar/zedcloud"
 	log "github.com/sirupsen/logrus"
-	"io"
-	"io/ioutil"
-	"net"
-	"net/http"
-	"os"
-	"strings"
-	"time"
 )
 
 const (
-	agentName       = "diag"
-	identityDirname = "/config"
-	selfRegFile     = identityDirname + "/self-register-failed"
-	serverFileName  = identityDirname + "/server"
-	deviceCertName  = identityDirname + "/device.cert.pem"
-	onboardCertName = identityDirname + "/onboard.cert.pem"
-	onboardKeyName  = identityDirname + "/onboard.key.pem"
```

Comment:
Only used in diag? But you might be missing changed to client.go??

Reply:
Good Catch.. Hmm.. Client.go has these defined relative to IdentityDir  And default value of IdentityDir is /config. But client has an option to pass on identityDir ( -D option ) that the user can theoretically set to something other than /config. But diag has this hardcoded to /config.

I am assuming we no longer need this flexibility to use different identityDirs.. and finalize on /config?

Reply:
Dropping the command line option to client is fine. I assume any testing of full agents will be done in a chroot/container when we get to that.

Reply:
Done.. Removed -D option from Client.go